### DISCUSSION TITLE
Organisation got flagged

### DISCUSSION THREAD

**Discussion by vrym2** [community/community#121665] [Repositories]
### Select Topic Area

General

### Body

My organization got flagged, and I opened a ticket. It has been three weeks, and no response yet! Please suggest an alternative to get my organization unflag!

**Comment by queenofcorgis** [comment]
Hi @vrym2, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
